src/test/java/org/folio/rest/api/HoldingsStorageTest.java

```diff
@@ -574,36 +569,7 @@ private JsonObject uprooted(UUID id) {
     tags.add("test-tag");
 
     return createInstanceRequest(id, "TEST", "Uprooted",
-      identifiers, contributors, UUID_TEXT, tags);
-  }
-
-  private JsonObject temeraire(UUID id) {
```

Comment:
Why has this been removed?

Reply:
It isn't used

Reply:
Ok, thanks. In future, please try to minimize the amount of extraneous changes within a pull request. 

Please feel free to issue preparatory pull requests that perform this kind of cleanup only.